### ISSUE TITLE
like instead of hide button

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ooskapenaar/awarenet#364]
```
I think there should be a like button in the notification under a blog instead 
of a hide button. Users will more likely like something and never really have 
reason to hide something, because there is not so much going on anyway. :-)
```

Original issue reported on code.google.com by `anna.wer...@gmail.com` on 22 Nov 2012 at 2:14

**Comment by AMWertlen** [comment]
I suggest we start with just very few options:
* user likes a single photo,
* user likes a single blog post,
* user likes a single discussion reply. 

The poster receives a notification in their feed that user likes the content.

If it makes things faster, nobody needs to see immediately how many people like the picture right after clicking, only after reload of page. And there need not be an option to unlike. 

I think that such an option would enhance activity for the benefit of awarenet and its users. Do you think you can do this @awarenetdev ?

**Comment by AMWertlen** [comment]
Won't do it now.
